trimmer_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative './decorator'
+
+class TrimmerDecorator < Decorator
+  def correct_name
+    @nameable.correct_name[0..10]
+  end
```

Comment:
- [ ] Good job so far 👏 I think there is a condition that you have not implemented here in the correct_name method of the TrimmerDecorator. Normally, the name should be trimmed if it's length is greater than 10.
Here is a hint on how it should look like 👇 
HINT:
```ruby
    def correct_name
      @nameable.correct_name.length > 10 ? @nameable.correct_name[0..9] : @nameable.correct_name
    end

```

Reply:
@BertrandConxy Thanks for feedback I appreciate it